src/mono/wasm/build/WasmApp.targets

```diff
@@ -30,6 +30,17 @@
       - $(WasmAssemblySearchPaths)          - used for resolving assembly dependencies
       - $(WasmSkipMissingAssemblies)        - Don't fail on any missing dependencies
 
+      - $(WasmGenerateAppBundle)            - Controls whether an app bundle should be generated.
+                                              Defaults to true.
+                                              This is useful for projects that want to handle their own
+                                              packaging, or app bundle generation, eg. Blazor.
+
+      - $(WasmStripAOTAssemblies)           - Whether to run `mono-cil-strip` on the assemblies.
```

Comment:
This seems to default to `$(RunAOTCompilation)` on L66, but `RunAOTCompilation` is said to default to `false` on L21.
Would `mono-cil-strip` be ran on assemblies that were not AOT compiled? I don't yet know the details of AOT compilation and stripping the CIL.

Reply:
It effectively defaults to true when `RunAOTCompilation=true`, because it is run only when we AOT assemblies.

Reply:
Because we have AOT'ed the code, we can strip all the method code from the assemblies, so that they are left with just the metadata.